pandas/indexes/base.py

```diff
@@ -1949,8 +1949,6 @@ def difference(self, other):
         theDiff = sorted(set(self) - set(other))
         return Index(theDiff, name=result_name)
 
-    diff = deprecate('diff', difference)
-
```

Comment:
hmm no tests on the deprecation to remove? docs (they r prob gone, but give a check)